pkg/config/config.go

```diff
@@ -607,6 +607,7 @@ func initConfig(config Config) {
 	config.SetKnown("system_probe_config.collect_local_dns")
 	config.SetKnown("system_probe_config.use_local_system_probe")
 	config.SetKnown("system_probe_config.enable_conntrack")
+	config.SetKnown("system_probe_config.enable_enobufs")
```

Comment:
this should be enable_enobuf

Reply:
sorry, I renamed everything to enobufs, is that OK?